### ISSUE TITLE
Annual recurring events not appearing on monthly calendar on some dates

### ISSUE THREAD

**Issue by gspice** [sugarcalendar/sugar-event-calendar#151]
First reported on this HS ticket:
https://secure.helpscout.net/conversation/722623828/95921?folderId=1909468

With this comment on the ticket:
```
Starts 12/09/2018 then every year on the 9th of Dezember // not working
Starts 06/17/2019 then every year on the 17th of Juni // working as expected
Starts 10/03/2019 then every year on the 3rd of Oktober // not working
Starts 04/25/2019 then every year on the 25th of April // working as expected
```

I tested the annual recurring date of `10/03/2019` and it doesn't appear on the next year's monthly view in October with these settings below.  I don't know it works on some dates and not others:

<img width="674" alt="screen shot 2018-12-08 at 19 31 44" src="https://user-images.githubusercontent.com/3018245/49692406-f521ef80-fb1f-11e8-9a35-3b718439f04d.png">

**Comment by JJJ** [comment]
This is being addressed in Sugar Calendar 2.0, and will not be fixed in the 1.x version.

**Comment by JJJ** [comment]
Closing in favor of #155. Thanks @gspice for the nudge though. 💚
